packages/belos/src/BelosSolverFactory_Belos.hpp

```diff
@@ -71,10 +71,18 @@ class SolverFactorySelector<double,MultiVec<double>,Operator<double>> {
 };
 
 #ifdef HAVE_TEUCHOS_COMPLEX
```

Comment:
We should at some point check the Trilinos-wide macros (if they exist) for this, rather than the Teuchos macros.  However, if the rest of Belos is using `HAVE_TEUCHOS_COMPLEX`, we should keep things consistent.